config/redis.yml

```diff
@@ -1,9 +1,9 @@
 development:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
+  port: <%= ENV['REDIS_PORT'] || 6379 %>
 test:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
+  port: <%= ENV['REDIS_PORT'] || 6379 %>
 production:
-  host: localhost
-  port: 6379
+  host: <%= ENV['REDIS_HOST'] || 'localhost' %>
```

Comment:
Nevermind this comment. This looks good to me. I'll create an issue for this if we want to make this better.

After looking into this our Redis initalizer does some silly things which doesn't make this work. And hardcodes the values from the yaml file. It will only set the different database.

Example:
```ruby
➜  HydraNorth2 ✗ export REDIS_URL=redis://10.0.1.1:6380/15
➜  HydraNorth2 ✗ echo $REDIS_URL
redis://10.0.1.1:6380/15
➜  HydraNorth2 ✗ rails c
2.4.0 :001 > Redis.current
 => #<Redis client v3.3.3 for redis://localhost:6379/15> 
2.4.0 :002 > exit
# MAKE CODE CHANGE TO FIX INITIALIZER
➜  HydraNorth2 ✗ rails c        
2.4.0 :001 > Redis.current
 => #<Redis client v3.3.3 for redis://10.0.1.1:6380/15> 
```